packages/stencil-component/src/components/read-along-component/readme.md

```diff
@@ -7,18 +7,21 @@
-| `pageScrolling`   | `page-scrolling`    | Toggles the page scrolling from horizontal to vertical. Defaults to horizontal                         | `"horizontal" \| "vertical"` | `"horizontal"` |
-| `svgOverlay`      | `svg-overlay`       | Overlay This is an SVG overlay to place over the progress bar                                          | `string`                     | `undefined`    |
-| `text`            | `text`              | The text as TEI                                                                                        | `string`                     | `undefined`    |
-| `theme`           | `theme`             | Theme to use: ['light', 'dark'] defaults to 'dark'                                                     | `string`                     | `'light'`      |
-| `useAssetsFolder` | `use-assets-folder` | Toggle the use of assets folder for resolving urls. Defaults to on to maintain backwards compatibility | `boolean`                    | `true`         |
+| Property             | Attribute                | Description                                                                                            | Type            | Default     |
+|----------------------|--------------------------|--------------------------------------------------------------------------------------------------------|-----------------|-------------|
+| `alignment`          | `alignment`              | The alignment as SMIL                                                                                  | `string`        | `undefined` |
+| `audio`              | `audio`                  | The audio file                                                                                         | `string`        | `undefined` |
+| `autoPauseEndOfPage` | `auto-pause-end-of-page` | Pause when at the end of a page                                                                        | `boolean`       | `false`     |
+| `cssUrl`             | `css-url`                | Optional custom Stylesheet to override defaults                                                        | `string`        | `undefined` |
+| `language`           | `language`               | Language  of the interface. In 639-3 code Options are - "eng" for English - "fra" for French           | `"eng" \        | "fra"`      | `'eng'`        |
+| `mode`               | `mode`                   | Choose mode of ReadAlong - either view (default) or edit                                               | `"EDIT" \       | "VIEW"`     | `"VIEW"`       |
+| `pageScrolling`      | `page-scrolling`         | Toggles the page scrolling from horizontal to vertical. Defaults to horizontal                         | `"horizontal" \ | "vertical"` | `"horizontal"` |
+| `scrollBehavior`     | `scroll-behavior`        |                                                                                                        | `"auto" \       | "smooth"`   | `"smooth"`     |
```

Comment:
Here too, all the types defined with an OR between multiple values don't render correctly on GitHub: https://github.com/ReadAlongs/Web-Component/tree/patch.page_transition/packages/stencil-component/src/components/read-along-component

Reply:
Strangely, these ones do render properly in VSCode's md preview, but those from the readme three directories up don't. I don't understand the different between them yet.

Reply:
fixed. The source had "eng" \   | "fra" so I removed the spaces between \ and | and it's fine now.